### DISCUSSION TITLE
Error with Excalidraw file delay; a long list of files with auto-saved PNG/SVG

### DISCUSSION THREAD

**Discussion by Missionary-OSBS** [zsviczian/obsidian-excalidraw-plugin#407] [Ideas]
Hi, 

I just used Excalidraw just a few days now. I found it very fascinating but ran into unexpected error. 

I had an Excalidraw drawing file, and during the editing process, I accidentally deleted the file using hotkey. Without knowing that, I could still edit and move stuff around. The file was in the trash but the canvas was still active, tricking me into thinking nothing ever happened. Later on, I came back to my drawing, and it's poof gone. Unfortunately for me, I emptied the trash. The solution would be when I delete a Excalidraw drawing file, it would be deleted immediately.

I found another solution of auto saving PNG file would be great. But at the same time, It would consume my storage if I had many drawings in my folder. Moreover, I want to separate Excalidraw file with the auto-saved PNG in different folder. The reason is that it would make repeated pattern of duplicated names and take up a long list of files. I would like to locate auto-saved PNG file into a specific different folder and that would be awesome. 

Thank you very much for creating this,
Missionary.

**Comment by zsviczian** [comment]
I think I’ve fixed this in the mean while. When you delete a file, the Excalidraw view should reflect that the file is no longer available.

Please in the future, open similar issues under issues and not discussion. I am not that very good at paying attention to the discussions, but I monitor issues on a daily basis.
